CHANGELOG.md

```diff
@@ -1,5 +1,54 @@
+ * Support `xml.format.preserveEmptyContent` setting with experimental formatter. See [#1242](https://github.com/eclipse/lemminx/issues/1242).
+ * Support `xml.format.joinCDATALines` setting with experimental formatter. See [#1241](https://github.com/eclipse/lemminx/issues/1241).
+ * Support `xml.format.preservedNewlines` setting with experimental formatter. See [#1240](https://github.com/eclipse/lemminx/issues/1240).
+ * Support `xml.format.enforceQuoteStyle` setting with experimental formatter. See [#1239](https://github.com/eclipse/lemminx/issues/1239).
+ * Support `xml.format.emptyElements` setting with experimental formatter. See [#1238](https://github.com/eclipse/lemminx/issues/1238).
+ * Provide comment formatting for experimental formatter. See [#1259](https://github.com/eclipse/lemminx/issues/1259). 
+ * CodeAction for cvc-complex-type.2.4.b: "insert all expected" vs. "insert all possible". See [#1255](https://github.com/eclipse/lemminx/issues/1255).  
+ * Suppress validation kinds based on file pattern. See [#1275](https://github.com/eclipse/lemminx/issues/1275).
+ * Report only XML syntax error for *.exsd files. See [#1308](https://github.com/eclipse/lemminx/pull/1308).  
+ * XML validation errors of "Language Servers" are only partially visible. See [#1145](https://github.com/eclipse/lemminx/issues/1145).
+ * Validate uri of XML catalog. See [#823](https://github.com/eclipse/lemminx/issues/823).
+ * Do not complete paths in attr unless beginning of value looks like a path. See [#1293](https://github.com/eclipse/lemminx/pull/1293). 
+ * Use the same proxy configuration for HTTPS. See [#1253](https://github.com/eclipse/lemminx/pull/1253).
+
+### Bug Fixes
+
+ * Fix tests on Windows OS. See [#1306](https://github.com/eclipse/lemminx/pull/1306).
+ * Annotations found in base complex type not shown on hover. See [#1268](https://github.com/eclipse/lemminx/pull/1268).
+ * Autocompletion of attribute values won't display documentation. See [#1260](https://github.com/eclipse/lemminx/pull/1260).
+ * Format CDATA sections without adding new lines. See [#1193](https://github.com/eclipse/lemminx/issues/1193).
+ * Formatting with "xml.format.emptyElements": "expand" chokes on malformed XML. See [#650](https://github.com/eclipse/lemminx/issues/650).
+ * Null pointer exception when processing code action cvc_complex_type_2_4_b. See [#1280](https://github.com/eclipse/lemminx/issues/1280).
+
+### Build
+
+ * Fix CI badge, add some more badges. See [#1262](https://github.com/eclipse/lemminx/pull/1262).
+ * Enable dependabot. See [#1309](https://github.com/eclipse/lemminx/pull/1309).
+ * Unit tests should not store data in the .lemminx folder under the user's home directory. See [#1265](https://github.com/eclipse/lemminx/issues/1265).
+ * Update Jenkins and GH Actions build scripts for branch renaming. See [#1261](https://github.com/eclipse/lemminx/pull/1261).
```

Comment:
Update Jenkins and GH Actions build scripts to reflect branch renaming.

Reply:
fixed